js/states.js

```diff
@@ -26,5 +26,6 @@ function updateStateBounds(error, stateBounds) {
     .data(stateBounds.features, function(d) {
       return d.properties.STATE_ABBV;
     })
-    .attr('d', buildPath);  
+    .attr('d', buildPath)
+    .on('click', zoomToFromState); //only clicks on mobile because of pointer-events
```

Comment:
i don't think we should need this `.on('click')` call in here because all elements of `#state-bounds` already exist (were created in `addStates`) and so should already have this method. You'll also note that there's a minor merge conflict with jordan's PR here; I think you may be able to adopt his code for this function wholesale.

this reminds me - i was worried about moving too much code earlier, but this might be an opportune time to move the `addStates` function into states.js, if you wouldn't mind.

Reply:
on second thought, I'm not going to move it because there are some global variables at the top of `map_utils.js` that will make this slightly more complex